SkyDrive/SkyDrive/Entities/BaseEntity.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SkyDrive.Entities
+{
+    public abstract class BaseEntity
```

Comment:
Why have you decided to use abstract class instead of interface?

Reply:
Because it's common property for all entities. Also it allows you to avoid duplicating code